src/routes/[lang=lang]/cadence-by-example/+page.ts

```diff
@@ -6,6 +6,12 @@ export const load = async ({ fetch, params }) => {
 		const response = await fetch(`/api/content/${params.lang}/cadenceByExample`);
 		const content = (await response.json()) as CadenceByExampleOverview[];
 
+		content.sort((a, b) => {
+			const prevNumber = Number(a.path.split('/').pop().split('-')[0]);
+			const nextNumber = Number(b.path.split('/').pop().split('-')[0]);
+			return prevNumber - nextNumber;
+		});
```

Comment:
This block is perfect! 
I know we already discussed this, but I just realized this would be better placed in the API layer.
This way, the API returns the Examples sorted correctly and we don't have to do it on the frontend and repeat ourselves each time we call this API.

Reply:
Totally agree! Is done